syne_tune/blackbox_repository/conversion_scripts/scripts/lcbench/lcbench.py

```diff
@@ -41,7 +41,7 @@
 
 RESOURCE_ATTR = "epoch"
 
-MAX_RESOURCE_LEVEL = 52
+MAX_RESOURCE_LEVEL = 50
```

Comment:
This naming is wrong. The max_resource_level is 51, right? It was wrong before as well, I suppose.

Reply:
The data has 52 epochs. Based on the notebook, the first epoch is before training started. Furthermore, they seem to discard the last epoch as well. The paper and the [repo](https://github.com/automl/LCBench) clearly state that they have 50 epochs. Therefore, I've dropped first and last data point which means we have 50 epochs in total. Does that make sense to you?

Reply:
Ah I see, you are right.